### DISCUSSION TITLE
Rivet with LLM Studio

### DISCUSSION THREAD

**Discussion by CiacioZ** [Ironclad/rivet#238] [Q&A]
I've tested revet with llmstudio but the chat node seems to be stuck. In the llm studio logs I see this:

[INFO] [LM STUDIO SERVER] Received OPTIONS request. {}
[ERROR] Unexpected endpoint or method. (POST /v1/chat/completition). Returning 200 anyway.

Is normal that rivet make "OPTION" request instead POST?

Should I set something to avoid this?

Thanks in advance :)

**Comment by abrenneke** [comment]
Yeah just enable CORS in LM Studio!

**Reply by CiacioZ** [reply]
Thank you for the answer but even with the CORS activated in LM Studio don't handle OPTION API call:

![lmstudio](https://github.com/Ironclad/rivet/assets/11212929/559b2e36-62eb-4b6a-bfab-8879898ddb9b)

Did I miss something?
